games-board/dreamchess/dreamchess-0.2.0.recipe

```diff
@@ -22,31 +22,42 @@ LICENSE="GNU GPL v3"
 REVISION="1"
 SOURCE_URI="https://github.com/dreamchess/dreamchess/releases/download/$portVersion/dreamchess-$portVersion.tar.gz"
 CHECKSUM_SHA256="7615699995a635c3dff6c952aa0d32723c415c8bb42448b4269d3daa199d8059"
+SOURCE_DIR="dreamchess-$portVersion"
+ADDITIONAL_FILES="dreamchess.rdef.in"
 
 ARCHITECTURES="x86_gcc2"
```

Comment:
Does it build under x86_64?

Reply:
I do not know, I do not have access to x86_64.
On x86_gcc2 it is built and works, on x86 it is built, but it works unstably.